accounts/models.py

```diff
@@ -6,6 +6,42 @@
 from phonenumber_field.modelfields import PhoneNumberField
 
 
+class School(models.Model):
+    """
+    Represents a school (ex: School of Arts & Sciences).
+    """
+
+    # boolean field for whether major is currently active/inactive
+    name = models.TextField(primary_key=True)
```

Comment:
This could probably just be a charfield since it won't be too many characters

Reply:
Oh, these also shouldn't be primary keys. Generally, we always want to keep the primary keys as auto-incrementing ids just because it makes any kind of migration easier when dealing with ints

Reply:
> This could probably just be a charfield since it won't be too many characters

fixed

Reply:
> Oh, these also shouldn't be primary keys. Generally, we always want to keep the primary keys as auto-incrementing ids just because it makes any kind of migration easier when dealing with ints

fixed